### ISSUE TITLE
Support for sys.config.src

### ISSUE THREAD

**Issue by afhole** [AdRoll/rebar3_format#245]
Is there any support for the config.src format?
When I run the formatter against sys.config.src it puts it the whole config onto a single line - is this format not supported, or do I need to tweak my config?

**Comment by elbrujohalcon** [comment]
🤔  Can you show me a sample config.src file? I was not aware of that kind of file existence at all.

**Comment by afhole** [comment]
I believe it is a Rebar3/Relx feature:

> Starting with Erlang/OTP 21 and rebar3 3.6.0 the configuration options sys_config_src and vm_args_src are available for explicitly including templates that will be rendered at runtime, substituting variables defined as ${VARIABLE} with their equivalent value in the shell environment.

http://rebar3.org/docs/deployment/releases/#environment-variable-replacement

**Comment by afhole** [comment]
I'm not sure how easy it would be to support, before substitution the files are not valid terms

**Comment by elbrujohalcon** [comment]
Yeah… Not easy at all.

**Comment by afhole** [comment]
Yeah maybe this is beyond the scope of a formatter, it's impossible to know even what the variables will expand to anyway

**Comment by elbrujohalcon** [comment]
It's not beyond the scope of the formatted (the formatter can format those files without expanding them) but it's beyond the current capabilities of OTP's `erl_syntax` and therefore handling it will require some serious adjustments to `ktn_dodger`.
